src/luascript.h

```diff
@@ -1543,7 +1557,7 @@ class LuaEnvironment : public LuaScriptInterface
 		LuaScriptInterface* getTestInterface();
 
 		Combat* getCombatObject(uint32_t id) const;
```

Comment:
getCombatObject should return Combat_ptr, and CombatSpell's combat property should be changed too, we will remove an old bug that made necessary to reload monsters when reloading spells and if that failed the server was closed since there was a dangling pointer(the combat inside CombatSpell)